src/Model/Command/UserMerge.php

```diff
@@ -77,4 +88,11 @@ protected function getCommandParameters(): array
             'source_user_id' => $this->sourceUserId,
         ];
     }
+
+    public function jsonSerialize(): array
```

Comment:
I'd not assert the values here. They should already be validated via custructor / setters.